src/main/java/core/basesyntax/strategy/QuantityIncrementer.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.mapvalidator.KeyPresentValidator;
+import java.util.Map;
+import java.util.function.BiFunction;
+
+public interface QuantityIncrementer extends KeyPresentValidator {
```

Comment:
As per the checklist, it's recommended to follow SOLID principles. The QuantityIncrementer interface extending KeyPresentValidator may violate the Interface Segregation Principle, as it forces implementors of QuantityIncrementer to also implement KeyPresentValidator, even if it's not needed for their use case.